lib/src/widgets/chat_list_widget.dart

```diff
@@ -272,19 +272,26 @@ class _ChatListWidgetState extends State<ChatListWidget>
                     onMoreTap: () {
                       _onChatListTap();
                       if (replyPopup?.onMoreTap != null) {
-                        replyPopup?.onMoreTap!();
+                        replyPopup?.onMoreTap!.call(
```

Comment:
The if check above seems unnecessary. Can you please instead use ? operator here to achieve the same?

Reply:
Please consider this for the entire file.